### ISSUE TITLE
Banana Cake Pop OAuth2 broke with V2

### ISSUE THREAD

**Issue by JamieBerghmans** [ChilliCream/graphql-platform#5471]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Describe the bug

I use the downloaded version of Banana Cake Pop, and was able to use OAuth2 perfectly. But ever since updating to BCP 2.0.0, my OAuth flow logs me in, and then throws "Either the parameter query or the parameter id has to be set"
Full error: `{"errors":[{"message":"Either the parameter query or the parameter id has to be set.","extensions":{"code":"HC0013"}}]}`

I tried downgrading back to BCP 1.0.0 but since today that has started throwing CORS errors so I assume something changed serverside?

The only issue I found related to this error is one where it's said that you should use the client and not the web browser for OAuth, but I am doing that.

### Steps to reproduce

The OAuth2 provider I'm using is Azure AD. Authenticating with OAuth2 with "Implicit" grant type, and "Access token" as response.

### Relevant log output

Error code show in popup window:
```json
{"errors":[{"message":"Either the parameter query or the parameter id has to be set.","extensions":{"code":"HC0013"}}]}
```
Error shown in dev console:
```shell
DOMException: Blocked a frame with origin "banana-cake-pop://app" from accessing a cross-origin frame.
    at banana-cake-pop://app/static/js/243.082d6152.chunk.js:1:114770
```

### Additional Context?

<img width="912" alt="image" src="https://user-images.githubusercontent.com/36448309/194549340-1f3e54a1-4368-4044-9ff6-b3764b2f898b.png">

### Product

Banana Cake Pop

### Version

2.0.0

**Comment by michaelstaib** [comment]
Hey,

thanks for reporting. The team is working on this issue and we will have this fixed in the insider build soon. I will report back as soon as the fix is merged.

**Comment by rstaib** [comment]
@JamieBerghmans could you then please try BCP version 1.0.4`?

**Comment by JamieBerghmans** [comment]
Version 1.0.4 works as expected :)

**Comment by rstaib** [comment]
> Version 1.0.4 works as expected :)

Perfect! At least you have a workaround for now 😉

**Comment by JamieBerghmans** [comment]
Yes! Thanks for the quick help! I'll patiently wait for the V2 fix now :)

**Comment by jhpetersen** [comment]
Run into the same issue after upgrading to BCP 2.
This is the second time OAuth authorization is killed with an BCP update - would it be possible to add an e2e test for this? 😉

**Comment by rstaib** [comment]
@jhpetersen this is due to a change in Electron. By the way we're now heading towards another approach in V3. Then it should work again 👍🏽

**Comment by rstaib** [comment]
@jhpetersen and sorry for any inconvenience.

**Comment by jhpetersen** [comment]
No worries, it's an open source project and i really appreciate your hard work on this! 🏅 👍
